docs/src/main/sphinx/connector/postgresql.rst

```diff
@@ -46,6 +46,53 @@ determine the user credentials for the connection, often a service user. You can
 use :doc:`secrets </security/secrets>` to avoid actual values in the catalog
 properties files.
 
+SQL support
+-----------
+
+The connector provides **read access and limited write access** to data and
+metadata in Delta Lake. The following section lists all supported SQL statements
+and provides further details.
+
+.. important::
+
+    No other SQL statements are supported.
+
+Read access:
+
+* :doc:`/sql/match-recognize`
```

Comment:
We realize that these are not all completely reliant on connector changes but they still provide read access and are relevant for connector users. Also without listing them here we can not make the note above work (No other SQL statements are supported)

Reply:
Technically any SQL clause (e.g. MATCH RECOGNIZE or CASE or LIMIT or OFFSET WITH TIES) works for all connectors by definition because the Engine implements them - the connector can only optimize them but it cannot prevent a SQL feature from working.

So MATCH RECOGNIZE shouldn't be here since it's part of the SELECT clause so as long as SELECT is supported MATCH RECOGNIZE is too.

cc: @martint Please correct if wrong.